synadm/cli/__init__.py

```diff
@@ -451,6 +462,11 @@ def get_redacted_token_prompt(cli_token):
             "Homeserver name (auto-retrieval or matrix.DOMAIN)",
             default=homeserver if homeserver else helper.config.get(
                 "homeserver", homeserver)),
+        "ssl_verify": click.prompt(
+            "Verify certificate (Default True, False allow to manage \
+                selfsigned certificate)",
```

Comment:
Please note that we have currently not defined the default values of the other config variables explicitly in the help string. Because if you run the configurator the current value (by default is set to the values defined in `APIHelper.CONFIG`) is presented behind each prompt within brackets, so the user exactly knows which value the corresponding config variable is currently set to. For example it looks like `Synapse base URL [http://localhost:8008]`. 
Therefore I don't think that there is a need to explicitly state the default value within the help string

Reply:
sounds good!